lib/components/Input/textarea/index.tsx

```diff
@@ -66,6 +67,8 @@ export const TextArea = forwardRef<HTMLTextAreaElement, TextAreaProps>(
                 ? statusIcon[status]
                 : rightIcon;
 
+        const id = uuid();
```

Comment:
can we use generateUniqueId here as well?

Reply:
oh yeah. missed this. updating it